### PR TITLE
[MUI-440] Update tablet mobile menu

### PR DESCRIPTION
**Pull request by slavaminterest**
# Description

Add component tablet mobile sidebar menu

## Jira ticket(optional)

[MUI-440](https://minterestfinance.atlassian.net/jira/software/projects/MUI/boards/7?assignee=621f2cc8c4d0fe006952986a&selectedIssue=MUI-440)

## Type of change

Please pick options that are relevant.

- [ ] Bug fix OR refactoring (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature or refactoring that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by DmytroMinterest — APPROVED** [review]

**Comment by DmytroMinterest** [comment]
Checked on temp UI-kit

File: src/components/SidebarMenu/SidebarMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  ListItem,
+  ListItemButton,
+} from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { CloseIcon } from 'assets/svg';
+
+import { Typography } from 'components';
+
+const DrawerStyled = styled('div')(({ theme }) => ({
+  width: '360px',
```
Comment (BerRossDolmat) [inline src/components/SidebarMenu/SidebarMenu.tsx:18]: magic number

File: src/components/SidebarMenu/SidebarMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    width: '130px',
+    backgroundColor: theme.palette.success.main,
+  },
+}));
+
+const TitleBlock = styled('div')({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  padding: '13.8px 25px',
```
Comment (BerRossDolmat) [inline src/components/SidebarMenu/SidebarMenu.tsx:45]: decimals in pixels

File: src/components/SidebarMenu/SidebarMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+      <DrawerStyled>
+        <TitleBlock>
+          <Typography text={title} variant='h2' />
+          <IconButton onClick={onClose}>
+            <CloseIcon />
+          </IconButton>
+        </TitleBlock>
+        <Divider />
+        <Box
+          sx={{ width: 250 }}
```
Comment (BerRossDolmat) [inline src/components/SidebarMenu/SidebarMenu.tsx:95]: magic number

File: src/theme.ts [hunk 4]
```diff
@@ -210,6 +213,12 @@ export const getTheme = (mode: PaletteMode): Theme =>
         }
       `,
       },
+      // MuiIconButton: {
```
Comment (BerRossDolmat) [inline src/theme.ts:216]: comment

File: src/components/SidebarMenu/SidebarMenu.tsx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+  title,
+  links,
+  BottomComponent,
+  onLinkClick,
+  isOpen,
+  onClose,
+  activeRoute,
+}: Props) => {
+  return (
+    // @ts-ignore
```
Comment (mr-teddy-bear) [inline src/components/SidebarMenu/SidebarMenu.tsx:79]: Why we need use ts-ignore? Maybe it is correct

File: src/components/SidebarMenu/SidebarMenu.tsx [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+        <Divider />
+        <Box
+          sx={{ width: 250 }}
+          role='presentation'
+          onClick={onClose}
+          onKeyDown={onClose}
+          className={'box'}
+        >
+          <List>
+            {links.map((link, index) => (
```
Comment (mr-teddy-bear) [inline src/components/SidebarMenu/SidebarMenu.tsx:102]: I am not sure, but using 'index' as a key in react it's not good practice(only my opinion) and you can skip this.

Comment (mr-teddy-bear) [reply src/components/SidebarMenu/SidebarMenu.tsx:102]: Done

File: src/components/SidebarMenu/SidebarMenu.tsx (hunk 5 above)
Comment (mr-teddy-bear) [reply src/components/SidebarMenu/SidebarMenu.tsx:79]: MUI issue

File: src/components/SidebarMenu/SidebarMenu.tsx (hunk 1 above)
Comment (mr-teddy-bear) [reply src/components/SidebarMenu/SidebarMenu.tsx:18]: What do you mean?

File: src/components/SidebarMenu/SidebarMenu.tsx (hunk 2 above)
Comment (mr-teddy-bear) [reply src/components/SidebarMenu/SidebarMenu.tsx]: Done

File: src/components/SidebarMenu/SidebarMenu.tsx (hunk 3 above)
Comment (mr-teddy-bear) [reply src/components/SidebarMenu/SidebarMenu.tsx:95]: What do you mean?

File: src/theme.ts (hunk 4 above)
Comment (mr-teddy-bear) [reply src/theme.ts]: Done

**Review by mr-teddy-bear — APPROVED** [review]

**Review by BerRossDolmat — APPROVED** [review]

**Review by DmytroMinterest — APPROVED** [review]

**Review by mr-teddy-bear — APPROVED** [review]

**Review by BerRossDolmat — APPROVED** [review]

**Pull request closed without merging**